chiplet/src/utilities.rs

```diff
@@ -168,10 +168,7 @@ impl<F: FieldExt> IsEqualChip<F> {
                 s_is_eq.clone() * (out.clone() * (one.clone() - out.clone())),
                 // if a != b then (a - b) * inverse(a - b) == 1 - out
                 // if a == b then (a - b) * 1 == 1 - out
-                s_is_eq.clone()
-                    * ((lhs.clone() - rhs.clone()) * delta_invert.clone() + (out - one.clone())),
-                // constrain delta_invert: (a - b) * inverse(a - b) must be 1 or 0
```

Comment:
is the constrain for delta_invert redundant?

Reply:
IMO, below line is redundant, which is already defined by above line.
"s_is_eq * (lhs.clone() - rhs.clone()) * ((lhs - rhs) * delta_invert - one)"

Reply:
> is the constrain for delta_invert redundant?

Oh, I saw the wrong line. There is no problem.